src/internal/extenders/utils.ts

```diff
@@ -0,0 +1,11 @@
+export function identity<T>(x: T): T {
+  return x;
+}
+
+export function isObject(obj: any) {
```

Comment:
[User-defined type guards](http://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) were made to be used in this file 😄
Also, [`unknown`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type) fits better here then `any`.